src/gui/properties/propertieswidget.ui

```diff
@@ -1017,14 +1017,14 @@
          <item>
           <widget class="QPushButton" name="selectAllButton">
            <property name="text">
-            <string>Select All</string>
+            <string>Check all items</string>
```

Comment:
Please delete this commit. These strings are there since the dawn of time. No one confused them so far. It is pretty clear on what they do.